### ISSUE TITLE
Move docker registry from quay.io to the docker registry.

### ISSUE THREAD

**Issue by tomgco** [yldio/paz#23]

**Comment by pierreozoux** [comment]
What are the reasons to move? *Just curious* :)

**Comment by tomgco** [comment]
Mainly due to: https://news.ycombinator.com/item?id=9119308, ideally by moving to the "official" registry it would make it easier to find (but that is just my opinion), public quay.io repos do not need auth so this could be a moot point. @lukebond has the reins on this.

**Comment by lukebond** [comment]
@tomgco yep that's right it's a moot point now that I've removed the instructions about credentials in the README. i see no reason why we can't store them in both registries, to be honest.

**Comment by tomgco** [comment]
Lets do that.
